profile/README.md

```diff
@@ -0,0 +1,28 @@
+<img align="right" width="33%" style="margin-bottom: 2em" src="http://owlbert.io/images/owlberts-png/Popper.psd.png">
+
+# Hi, we're ReadMe! <img height=36 align=center src=https://cliply.co/wp-content/uploads/2019/06/391906110_WAVING_HAND_400px.gif>
+
+Welcome to our GitHub org! [At ReadMe][readme], we want developers to succeed. That's why building thoughtful developer experiences is at the core of everything we do.
+
+### And by “developer”, we mean a _lot_ of people…
+
+:memo:   [Anyone that builds API docs](https://readme.com/customers) on ReadMe. <br>
+:monocle_face:   People that use API docs built on ReadMe. <br>
+:floppy_disk:   Folks that use any of our open source libraries.[^come-work-with-us] <br>
```

Comment:
Linking to some of our open source libraries here might be nice!

Reply:
We did link to some in an initial draft but since this is going to be showing on [our GitHub profile](https://github.com/readmeio), they'll be able to see our pinned repositories right below this blurb and I feel like that's enough? Especially since our pins and our featured repos may change over time!

Reply:
Ooh fair enough! Agreed!